Src/PerformanceCollector/Shared/QuickPulseTelemetryModule.cs

```diff
@@ -120,6 +124,12 @@ public QuickPulseTelemetryModule()
         public bool DisableTopCpuProcesses { get; set; }
 
         /// <summary>
+        /// Gets the auth API key.
```

Comment:
>Gets the auth API key. [](start = 12, length = 22)

Can you please elaborate on how it is used and where to obtain it as this is a public contract for our users